CODEOWNERS

```diff
@@ -1 +1,2 @@
-
+# Owner: https://github.com/ckstettler
+* @CaptianQwirk
```

Comment:
I gave you the incorrect name:  It should be CaptainQuirk

Reply:
should be good to go -- lmk if otherwise.